IMPEDANCE_FILTER:
  M: [25000, 250, 25000, 0.0, 0.0, 0.0]
  B: [100000, 1000, 100000, 0.0, 0.0, 0.0]
  K: [100000, 1000, 100000, 0.0 , 0.0, 0.0]
  dead_zone: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
MRAC:
  REFERENCE_MODEL:
    OMEGA: [3.16227766, 3.16227766, 3.16227766, 1.054092553, 1.054092553, 1.054092553]
    ZETA: [1, 1, 1, 1, 1, 1]
  WEIGHTING_FACTORS:
    Wp: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    Wd: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  INTEGRAL_ADAPTATION_GAINS:
    a1: [0.2, 0.2, 0.0, 0.04, 0.04, 0.0]
    b1: [0.1, 0.1, 0.0000, 0.00003, 0.00003, 0.0]
    c1: [0.1, 0.1, 0.0000000, 0.000000005, 0.000000005, 0.0]
  PROPORTIONAL_ADAPTATION_GAINS:
    a2: [0.0, 0.0, 0.00, 0.004, 0.004, 0.0]
    b2: [0.0, 0.0, 0.0000, 0.00003, 0.00003, 0.0]
    c2: [0.0, 0.0, 0.0000000, 0.000000005, 0.000000005, 0.0]
  SIGMA_MODIFICATION_GAINS:
    sigma1: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    sigma2: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    sigma3: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  INITIAL_GAINS:
    Kp: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    Kd: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
AIC:
  INTEGRAL_ADAPTATION_GAINS:
    ke1: [0.0, 0.005, 0.000, 0.0, 0.0, 0.0]
  PROPORTIONAL_ADAPTATION_GAINS:
    ke2: [0.0, 0.015, 0.0, 0.0, 0.0, 0.0]
  INITIAL_GAINS:
    Ke: [0.0, 0.000, 0.000, 0.0, 0.0, 0.0]
  WEIGHTING_FACTORS:
    Wp: [0.0, 1.5, 0.0, 0.0, 0.0, 0.0]
    Wd: [0.0, 0.8, 0.0, 0.0, 0.0, 0.0]